synfig-studio/src/synfigapp/actions/valuedesclink.cpp

```diff
@@ -44,11 +44,10 @@
 #include <synfig/blinepoint.h>
 
 #include <synfigapp/localization.h>
-#include <zconf.h>
+//#include <zconf.h>
```

Comment:
What is this header doing here anyway?

Reply:
This header was added by this commit (https://github.com/synfig/synfig/commit/264825e7c5e730e4611ccfa916ca6188811777ee), apparently by mistake